crypto/dh/dh_gen.c

```diff
@@ -219,6 +220,9 @@ static int dh_builtin_genparams(DH *ret, int prime_len, int generator,
         goto err;
     if (!BN_set_word(ret->params.g, g))
         goto err;
+    /* We are using safe prime p, set key length equivalent to RFC 7919 */
+    ret->length = (2 * ossl_ifc_ffc_compute_security_bits(prime_len)
+                   + 24) / 25 * 25;
```

Comment:
I see this being tested in PKCS3 files. Does this get used by DSA? I see no DSA tests. Do we have other formats in which we can save DH parameters?

Reply:
It is not used by DSA.